doc/templating.rst

```diff
@@ -202,8 +202,24 @@ Overriding works like this::
 
 .. data:: css_files
 
-   Similar to :data:`script_files`, for CSS files.
+   Similar to :data:`script_files`, for CSS files::
 
+      {% set css_files = css_files + ["_static/mystyle.css"] %}
```

Comment:
IMO, rewriting a template variable is bad manner.
Even if you or someone want to use such workaround, official document should not provide such information.